README.md

```diff
@@ -80,3 +80,18 @@ This patch includes the following changes:
   [ddvk's Qt Plugin for Lamy Pen](https://github.com/ddvk/remarkable-stylus) first. You will not need to install ddvk-hacks like it recommends, as this replaces that dependency.
 - The toolbar icon changes from the default open icon to a brush icon.
 - The close icon (x) in the top right corner has been removed.
+
+## License
+
+This repository is provided under the MIT License, as described in the
+root `LICENSE` file. However, please note that all the files in the "patches"
+directory are marked as "All Rights Reserved," which means that they are
+protected by copyright, and no specific license information is available for
+them. Users should assume that they have no rights or permissions to use,
```

Comment:
This means that without express "permission to use [the patches]", people cannot install your bundle of hacks.

Also the phrasing - "should assume that" - is odd. 
I suggest